src/display/xfa_layer.js

```diff
@@ -161,13 +161,22 @@ class XfaLayer {
       }
     }
 
+    /*
+       TODO: re-enable that stuff once we've JS implementation.
+       for (const el of rootDiv.querySelectorAll(
+       ".xfaDisabled input, .xfaDisabled textarea"
+       )) {
+       el.setAttribute("disabled", true);
+       }
+       for (const el of rootDiv.querySelectorAll(
+       ".xfaReadOnly input, .xfaReadOnly textarea"
+       )) {
+       el.setAttribute("readOnly", true);
+       }
+     */
```

Comment:
Nit: Please format block comments as follows
```javascript
/**
 * Comment here...
 */
```

Reply:
Could we add a rule in the linter for that ? (personally I always thought that this kind of comment (the ones you mentioned) were for function documentation (javadoc style)).

Reply:
> Could we add a rule in the linter for that ?

I'll try to look into that later, when I've got some spare time.

>  (personally I always thought that this kind of comment (the ones you mentioned) were for function documentation (javadoc style)).

You're correct that we mostly use that kind of comments for JSDocs, and *usually* multi-line comments in the code-base are formatted as:
```javascript
// This is a 
// multi-line
// comment.
```

My point, purely for consistency, was that the *particular* comment formatting used in the earlier version of this patch looked a little "uncommon" for the code-base :-)